Gemfile

```diff
@@ -13,7 +13,7 @@ gem 'rest-client'
 gem 'exercism-config', '>= 0.76.0'
 
 gem 'rubyzip'
-gem 'rugged' # Git
+gem 'rugged', '1.4.2' # Git
```

Comment:
Should we do the same for the website? To prevent us from forgetting that we can't just upgrade it.

Reply:
Yes please.

Reply:
[PR opened](https://github.com/exercism/website/pull/2914/)